Non standard properties FAQ (#771)
rudderlabs/rudderstack-docs: docs/destinations/warehouse-destinations/warehouse-schema.mdx

| `a21620be-6502-44d6-941d-78209a386d58` | `add_to_cart` | `price`     | `NA`         |
| `1e42b2b3-8b6a-49da-8502-83a8db334375` | `add_to_cart` | `added_at`  | `05/25/2020` |

## FAQ

### Can I change the namespace \(schema name\) of my data warehouse in RudderStack?

Yes, you can. Although the default namespace will be the source name, RudderStack gives you the option to explicitly set the namespace while setting up your warehouse destination. For more information, refer to the <Link to="/destinations/warehouse-destinations/">warehouse-specific destination settings</Link> for configuring the namespace in the RudderStack dashboard.

### Why am I not able to see the properties added at the top level of an event in warehouse destination?

> RudderStack drops the non-standard properties (any property apart from the <Link to="#standard-rudderstack-properties">standard properties</Link>) declared at the top level of an event. However, it allows you to add such properties in the `context` or `properties` section of the event payload.

Reviewer: ```suggestion
RudderStack drops any non-standard properties (properties apart from the <Link to="#standard-rudderstack-properties">standard properties</Link>) declared at the top level of an event. However, you can add such properties in the `context` or `properties` section of the event payload.
```
